src/main/java/io/cryostat/agent/remote/RecordingsContext.java

```diff
@@ -231,146 +214,112 @@ private void handleStartRecordingOrSnapshot(HttpExchange exchange) throws IOExce
             try (OutputStream response = exchange.getResponseBody()) {
                 mapper.writeValue(response, recording);
             }
-        } catch (QuantityConversionException
-                | ServiceNotAvailableException
-                | FlightRecorderException
-                | org.openjdk.jmc.rjmx.services.jfr.FlightRecorderException
-                | InvalidXmlException
-                | IOException e) {
+        } catch (IOException e) {
             log.error("Failed to start recording", e);
             exchange.sendResponseHeaders(HttpStatus.SC_INTERNAL_SERVER_ERROR, BODY_LENGTH_NONE);
         } catch (InvalidEventTemplateException e) {
+            log.warn("Invalid custom event template", e);
             exchange.sendResponseHeaders(HttpStatus.SC_BAD_REQUEST, BODY_LENGTH_NONE);
         }
     }
 
-    private void handleStopOrUpdate(HttpExchange exchange, long id) throws IOException {
+    private void handleStopOrUpdate(HttpExchange exchange, long recordingId) throws IOException {
```

Comment:
Is there a reason `id` was changed to `recordingId` here? I noticed it wasn't in `handleGetRecording` and `handleDelete`

Reply:
I don't recall any particular reason, I guess this is just something I did while refactoring and cleaning up.